.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Short description
+I have:
+- [ ] There is a corresponding issue for this pull request.
+- [ ] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
+- [ ] There is only strictly only one commit per issue.
```

Comment:
I do not know if you really want to have this last line. 
I know it is copied from loklak. For small fixes, it may not be required to create an issue and I do not like to turn away people because of administration overhead.